packages/teleport/src/Desktops/useDesktops.ts

```diff
@@ -26,11 +26,17 @@ export default function useDesktops(ctx: Ctx) {
   const { attempt, run } = useAttempt('processing');
   const { clusterId } = useStickyClusterId();
   const username = ctx.storeUser.state.username;
+  const windowsLogins = ctx.storeUser.getWindowsLogins();
 
   const [searchValue, setSearchValue] = useState<string>('');
 
   const [desktops, setDesktops] = useState<Desktop[]>([]);
 
+  const getWindowsLoginOptions = useCallback(
```

Comment:
Are you certain that `useCallback` is necessary here?

Reply:
not needed per alexey (could we also remove it from `useNodes`)